docs/multicluster/user-guide.md

```diff
@@ -689,7 +599,7 @@ spec:
       toServices:
         - name: secured-service   # an exported Multi-cluster Service
           namespace: svcNamespace
-          scope: clusterSet
+          scope: ClusterSet
```

Comment:
I got an issue report from a user, seems only `clusterSet` works for `toServices`, and `ClusterSet` won't take any affect. I take a quick look at the code, the field is string without enum value. @GraysonWu  @Dyanngg  could you take a look? I believe we need another fix before we merge this.

Reply:
Let us change to enum and backport the fix to 1.10.

Reply:
Sure. PR #4562 opened.

Reply:
The new commit has been cherry-picked in this PR. @jianjuns could you take a look again? thanks.